pages/en/contrib/code-contrib.md

```diff
@@ -1,17 +1,109 @@
+
+Also, please read the source code before contributing. If something is confusing, poorly documented, poorly commented, or could otherwise be improved: open a GitHub issue and link to the file and line of code with a useful description of what you would like to see or think is missing.
+
+### Recommended Node.js version for code contributions
+
+Code contributions should only be using features and language constructs available in v0.10.
+
+{% include note.html content="Check your Node.js version with the command `node -v`.
+" %}
+
+At this time, we are still supporting Node.js v0.10 and do not transpile. We would like to change this in the near future (ie. drop support for versions older than 4/6 or transpile), but extra caution is required as this update would affect a large number of users. Thanks for your patience.
+
+## Submitting a pull request
+
+Follow the [GitHub Flow](http://scottchacon.com/2011/08/31/github-flow.html).
+
+ 1. Create a GitHub issue for large changes and discuss the change there before
+    coding. You can skip this step and submit the pull request for minor
+    changes.
+ 1. Fork the repository on GitHub.
+ 1. Create a branch for you change/feature off of the master branch.
+ 1. Make your change. Remember to update tests as well as code! Always
+    run all the tests to assure nothing else was accidentally broken. For bugs, adding a failing test and submitting a pull request usually leads to the bug being fixed quickly. For features, include tests that cover the entire feature. For remote / client facing features, include integration tests in the [loopback-example-app](](https://github.com/strongloop/loopback-example-app).
+ 1. Check for unnecessary whitespace with `git diff --check` before committing.
+ 1. Make commits of logical units and push them to Github.
+ 1. Use a descriptive commit message, and follow
+    [50/72 format](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
+ 1. Use GitHub's pull requests to submit the patch. Feel free to issue the pull
+    request as soon as you have something partial to show and get reviewed.
+    You can add more commits to your pull request as you are progressing
+    on the implementation.  The title of the pull request should start
+    with "WIP" for "work in progress" when the pull request is not complete
+    yet.
+ 1. Request a code review. Add a specific committer to speed up this process. (for example, `@bajtos`, `@raymondfeng`).
+ 1. Make any requested changes and push to your fork. Make sure your changes are still based on the latest code (use `git rebase upstream/master`).
+
+## Running tests
+
+All LoopBack projects follow the same convention for running tests. To run a projects test suite, change to the project's directory and run:
+
+```sh
+npm test
+```
+
+If a test is failing, you should open an issue on the appropriate repositories GitHub issues page.
+
+#### Signing patches
```

Comment:
Signing the patches is no longer needed. We are collecting CLAs through the clahub app. This whole section can be removed (or replaced with text describing how to sign the CLA through the web app).